WORKSPACE

```diff
@@ -151,13 +151,20 @@ load("@io_bazel_rules_d//d:d.bzl", "d_repositories")
 
 d_repositories()
 
-http_archive(
+# Until rules_nodejs is updated
+git_repository(
```

Comment:
Could we keep http_archive rule instead of git_repository rule?

In [git_repository's documentation](Warning: this rule has several limitations. We recommend using http_archive instead for more robust and secure performance.)
> Warning: this rule has several limitations. We recommend using http_archive instead for more robust and secure performance.

Reply:
Yeah, no problem. I had it there only until the rules_nodejs was actually tagged. I'll make a new PR with just the formatting changes. The CI fails without them, that's why I added them.